lib/libutee/sub.mk

```diff
@@ -12,6 +12,8 @@ srcs-y += tee_api_objects.c
 srcs-y += tee_api_operations.c
 srcs-y += tee_api_se.c
 srcs-y += tee_api_panic.c
+srcs-y += tee_tcpudp_socket.c
+srcs-y += tee_socket_pta.c
```

Comment:
This can be compiled with or without GP sockets support enabled in the rest of the system. An advantage with that is that the library is the same regardless of core configuration.